src/SpotifyArduino.cpp

```diff
@@ -20,142 +20,70 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA
 
 #include "SpotifyArduino.h"
 
-SpotifyArduino::SpotifyArduino(Client &client)
+SpotifyArduino::SpotifyArduino(WiFiClient &client)
```

Comment:
The reason for "Client" versus "WifiClient" is it keeps the library generic so it can be used for other architectures than just the esp32/esp8266 so it in theory works with wifi Nina boards from Arduino and potentially pico w, and whatever other boards come down the line.